modules/i3s/src/lib/parsers/parse-i3s-attribute.js

```diff
@@ -9,11 +9,12 @@ import {STRING_ATTRIBUTE_TYPE, OBJECT_ID_ATTRIBUTE_TYPE, FLOAT_64_TYPE} from './
  */
 export async function parseI3STileAttribute(arrayBuffer, options) {
   const {attributeName, attributeType} = options;
-  if (!attributeName || !attributeType) {
+
```

Comment:
Comment that this handles a case not covered in spec?

@Tamrat-B

Reply:
@Ib To be honest in [spec](https://github.com/Esri/i3s-spec/blob/master/docs/1.7/value.cmn.md) if `attrbuteValues` exists then `valueType` property is mandatory. In our case `valueType` exists. But I am not sure about its value. Maybe it can be empty. It is not described in the spec.
I think we always should show all attributes from `attributeStorageData`  to user even them don't have values.

Reply:
Yes always showing all fields sounds reasonable. It is like a database record I suppose, values can be empty.

The application can choose to filter out empty fields if that makes sense for the app.